pages/registration-summary.njk

```diff
@@ -97,11 +97,11 @@ summaryListDeclarationHelper %}
           {%elif props.cumulativeFullAnswers.registration_role === props.enums.operatorType.PARTNERSHIP.key %}
           {% set html %}
           <div>
-            <div id="main_partner_primary_number">{{ propsData.main_partner_primary_number if propsData.main_partner_primary_number else "" }}</div>
-            <div id="main_partner_secondary_number">{{ propsData.main_partner_secondary_number if propsData.main_partner_secondary_number else "" }}</div>
+            <div id="operator_primary_number">{{ propsData.operator_primary_number if propsData.operator_primary_number else "" }}</div>
```

Comment:
This will work but you don't actually need to duplicate the html as it is the same as for non-partnerships now...
Could instead move lines 90-95 outside the if statement to line 88, then it can be used for both PERSON and PARTNERSHIPs.  Let me know if that doesn't make sense...